package.json

```diff
@@ -23,21 +23,21 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@sentry/node": "5.30.0",
-    "algoliasearch": "3.35.1",
+    "@sentry/node": "6.5.1",
+    "algoliasearch": "4.9.1",
```

Comment:
aren't there breaking changes in the algolia usage?

Reply:
well you are better suited to answer to that, but the basic client creation and save objects seems to work fine.
Note I tested locally with bootstrap with 100 Packages.

Reply:
If it was typescript we could find more of the errors, but for example:

```
async function putDefaultSettings(index, config) {
  await index.setSettings(config.indexSettings);

  await index.batchSynonyms(config.indexSynonyms, {
    replaceExistingSynonyms: true,
  });
  const { taskID } = await index.batchRules(config.indexRules, {
    replaceExistingRules: true,
  });

  await index.waitTask(taskID);
}
```

must become:

```
  await index.setSettings(config.indexSettings);

  await index.saveSynonyms(config.indexSynonyms, {
    replaceExistingSynonyms: true,
  });
  const { taskID } = await index.saveRules(config.indexRules, {
    replaceExistingRules: true,
  });

  await index.waitTask(taskID);
}
```

Reply:
ah good catch, this one is executed only if the original index does not exists. I missed it 👌🏻 
in the next PRs I'll try to create integration test, a lot of things are untested right now, too easy to make mistake (and I do a lot of them ahah)

Reply:
![Screen Shot 2021-06-04 at 17 21 09](https://user-images.githubusercontent.com/1637651/120825095-555f4400-c559-11eb-8a16-986d08476886.png)